src/platform_impl/macos/view.rs

```diff
@@ -325,19 +329,22 @@ extern "C" fn frame_did_change(this: &Object, _sel: Sel, _event: id) {
         let state_ptr: *mut c_void = *this.get_ivar("winitState");
         let state = &mut *(state_ptr as *mut ViewState);
 
-        if let Some(tracking_rect) = state.tracking_rect.take() {
-            let _: () = msg_send![this, removeTrackingRect: tracking_rect];
+        if let Some(tracking_area) = state.tracking_area.take() {
+            let _: () = msg_send![this, removeTrackingArea: tracking_area];
```

Comment:
Since you are allocation a new tracking area later, the old tracking area here should be freed by sending a [`release`](https://developer.apple.com/documentation/objectivec/1418956-nsobject/1571957-release) message to it.